auth/oauth/oauth.go

```diff
@@ -10,9 +10,18 @@ import (
 	"golang.org/x/oauth2"
 )
 
-const (
-	azureTenantId = "4a67d088-db5c-48f1-9ff2-0aace800ae68"
-)
+type AzureTenant struct {
+	DnsZone            string
+	AzureApplicationID string
+}
```

Comment:
Is this type used anywhere?

Reply:
No,  just removed it. Good catch.
At first I wanted to use it to replace the: 
`type CloudType int`
But I noticed that the InferCloudFromHost func in previous versions would be unstable. So I handled it separately.